tests/E2E/CliTest.php

```diff
@@ -98,6 +98,22 @@ public function test_bug_yield(): void
         $this->assertStringContainsString($expectedErrors, $process->getOutput());
     }
 
+    /**
+     * @group 8.x
```

Comment:
phpunit has a way to exclude tests based on phpunit version: https://phpunit.readthedocs.io/en/7.3/incomplete-and-skipped-tests.html#skipping-tests-using-requires

This would result in a simpler build configuration, since we could avoid checking for php version here: `run: ./bin/phpunit ${{ matrix.php-versions == '8.0' && ' --exclude-group 8.x' || '' }}`

Reply:
Right! I can try to use this approach.